etc/pip-requires

```diff
@@ -16,3 +16,8 @@ paramiko==2.7.2                                   # SSH2 protocol library
 Flask==1.1.2                                      # Python web framework
 oic==1.2.1                                        # for authentication via OpenID Connect protocol
 prometheus_client==0.9.0                          # Python client for the Prometheus monitoring system
+python-magic>=0.4.15,<0.5.0                       # File type identification using libmagic
```

Comment:
Where do we use this?

Reply:
This might come in as a secondary dependency. In general, we should remove all entries that are not directly needed.

Reply:
I looked this up, it is used by the auditor to identify mimetypes. I guess we need to keep this.

Reply:
Just saw that @jamesp-epcc commented this as well :-)